test/apiv2/40-pods.at

```diff
@@ -52,11 +61,16 @@ t POST "libpod/pods/fakename/unpause" 404\
   .cause="no such pod" \
   .message="no pod with name or ID fakename found: no such pod"
 
+t DELETE libpod/pods/foo?force=false 500 \
+  .cause~".*cannot remove container .* as it is running.*"
```

Comment:
Playing around some more, it looks like `message` and `cause` are identical. If that is the intention, you may want to duplicate line 64, `s/cause/message/`. (Backslashing line 64 of course). You may also want to add as much or little other context as you wish, e.g.
```
   .cause~"not all containers could be removed from pod.*cannot remove container .* as it is running - running or paused containers blah blah blah you get my point"
```
(Might as well wait to see how CI goes though)